manifests/mod/dav_svn.pp

```diff
@@ -13,7 +13,7 @@
 
   ::apache::mod { 'dav_svn': }
 
-  if $::osfamily == 'Debian' and ($::operatingsystemmajrelease != '6' and $::operatingsystemmajrelease != '10.04' and $::operatingsystemrelease != '10.04' and $::operatingsystemmajrelease != '16.04') {
+  if $::osfamily == 'Debian' and ($::operatingsystemmajrelease != '6' and !($::operatingsystemmajrelease in ['10.04', '16.04', '18.04'])) {
```

Comment:
We have long dropped support for 10.04 so you can remove it here :)

Reply:
removed it